game.h

```diff
@@ -15,6 +15,9 @@ class Game {
   // This function does not return because the game loop is inside.
   bool Run();
 
+  // Limits the frame rate.
+  void LimitFrames();
```

Comment:
Also, this should be a private function since it is only used internally by the game class itself (it isn't part of the game class public API).